Tests/Provider/VimeoProviderTest.php

```diff
@@ -97,6 +97,11 @@ public function testThumbnail()
         $provider->generateThumbnails($media);
 
         $this->assertEquals('default/0011/24/thumb_1023457_big.jpg', $provider->generatePrivateUrl($media, 'big'));
+
+        $provider->preRemove($media);
+        $this->assertTrue(in_array('default/0011/24/thumb_1023457_reference.jpg', $media->getAssetPaths()), 'Media::getAssetPaths() return the correct paths to delete');
+        $this->assertTrue(in_array('default/0011/24/thumb_1023457_big.jpg', $media->getAssetPaths()), 'Media::getAssetPaths() return the correct paths to delete');
```

Comment:
maybe we should assert, that `$media->getAssetPaths() is equal to our testArray`, because otherwise there could be more paths in `$media->getAssetPaths()` or asssert the count == 2